common/CMakeLists.txt

```diff
@@ -12,19 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.10 FATAL_ERROR)
-FIND_PACKAGE(PkgConfig REQUIRED)
-
 OPTION(BUILD_TRUSTED "Build modules for running inside an enclave" ON)
 OPTION(BUILD_UNTRUSTED "Build modules for running with SGX outside an enclave" ON)
 OPTION(BUILD_CLIENT "Build modules for running clients without SGX" OFF)
 OPTION(BLOCK_STORE_DEBUG "Debug logging for block store operations" OFF)
 
-INCLUDE(CMakeVariables.txt)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.10 FATAL_ERROR)
```

Comment:
This might open a whole other can of worms but... should we upgrade to a higher cmake version while we're at it? 3.22 is the latest on 20.04 and 22.04.

Reply:
we can update this. though i don't think we're using any features from newer versions.

Reply:
opened #397 to track this.